src/rpc/stats.h

```diff
@@ -54,30 +54,36 @@ struct RPCStats {
 class CRPCStats
 {
 private:
+    std::atomic_bool lock_stats{false};
     std::map<std::string, RPCStats> map;
-public:
     bool active{DEFAULT_RPC_STATS};
 
+public:
+    bool isActive() {
+        return active;
```

Comment:
This needs to be guarded too. But can be granular. Use another atomic bool, and return. Same way for set.

Reply:
`return active.load();`